### ISSUE TITLE
Prisma Studio not showing data from database upon launch

### ISSUE THREAD

**Issue by Raamzeez** [prisma/studio#649]
1. Prisma version (`prisma -v` or `npx prisma -v`)

```
Environment variables loaded from .env
prisma               : 2.20.1
@prisma/client       : 2.21.0-dev.6
Current platform     : darwin
Query Engine         : query-engine 60ba6551f29b17d7d6ce479e5733c70d9c00860e (at node_modules/@prisma/engines/query-engine-darwin)
Migration Engine     : migration-engine-cli 60ba6551f29b17d7d6ce479e5733c70d9c00860e (at node_modules/@prisma/engines/migration-engine-darwin)
Introspection Engine : introspection-core 60ba6551f29b17d7d6ce479e5733c70d9c00860e (at node_modules/@prisma/engines/introspection-engine-darwin)
Format Binary        : prisma-fmt 60ba6551f29b17d7d6ce479e5733c70d9c00860e (at node_modules/@prisma/engines/prisma-fmt-darwin)
Default Engines Hash : 60ba6551f29b17d7d6ce479e5733c70d9c00860e
Studio               : 0.365.0
```

2. Logs from Developer Tools Console or Command line, if any:
```
Studio version 0.359.0    main.js:402
Bootstrap Duration: 1144.985107421875 ms    main.js:402
DevTools failed to load SourceMap: Could not load content for file:///Applications/Prisma Studio.app/Contents/Resources/app.asar/build/preload.js.map: 
DevTools failed to load SourceMap: Could not load content for file:///Applications/Prisma%20Studio.app/Contents/Resources/app.asar/renderer/main.js.map: 
DevTools failed to load SourceMap: Could not load content for file:///Applications/Prisma%20Studio.app/Contents/Resources/app.asar/renderer/main.d5b804b401f9a0033fdb.css.map: 
```

3. Does the issue persist even after updating to the latest `prisma` alpha? (`npm i -D @prisma/cli@dev`)
```
Yes
```

When I run `npx prisma studio` or open the Prisma Studio Desktop Application, my data from my postgresql database is not shown immediately. I have to click on a model, then interact with the table somehow by doing something like clicking the refresh button or sorting a column, and then the rows will populate with the corresponding information for that table. However, the other tables are still left empty until I do the same thing again for each model. 

Expected Behavior: The data from my database to be shown immediately upon opening a model

**Comment by madebysid** [comment]
@Raamzeez I think I've seen this too recently. Strange since I don't think I changed anything with it. I'll take a look.

**Comment by madebysid** [comment]
@Raamzeez I haven't been able to reliably reproduce this. Can you check if this is still an issue with tomorrow's Prisma release?

**Comment by Raamzeez** [comment]
@madebysid Yes I will check tomorrow evening and see if it starts showing the data upon launch. I'll let you know what happens.

**Comment by Raamzeez** [comment]
@madebysid Apologies for the late response; I got caught up in a few other things and had forgotten about the issue until I started using Prisma Studio again recently. When I open up Prisma Studio right now, it populates data for a few of my models (an improvement from before), but not all of them. I did upgrade my prisma version since I opened the issue.

Prisma version (prisma -v or npx prisma -v): 
`
prisma               : 2.21.2
@prisma/client       : 2.21.0-dev.6
Current platform     : darwin
Query Engine         : query-engine e421996c87d5f3c8f7eeadd502d4ad402c89464d (at node_modules/prisma/node_modules/@prisma/engines/query-engine-darwin)
Migration Engine     : migration-engine-cli e421996c87d5f3c8f7eeadd502d4ad402c89464d (at node_modules/prisma/node_modules/@prisma/engines/migration-engine-darwin)
Introspection Engine : introspection-core e421996c87d5f3c8f7eeadd502d4ad402c89464d (at node_modules/prisma/node_modules/@prisma/engines/introspection-engine-darwin)
Format Binary        : prisma-fmt e421996c87d5f3c8f7eeadd502d4ad402c89464d (at node_modules/prisma/node_modules/@prisma/engines/prisma-fmt-darwin)
Default Engines Hash : e421996c87d5f3c8f7eeadd502d4ad402c89464d
Studio               : 0.371.0`

**Comment by madebysid** [comment]
Hey @Raamzeez, I've made a fix on `prisma@2.23.0-dev.24` that should fix this. Could you try updating and give it another shot?
If it works, then this fix shuld land on stable next week. If not, please let me know! I understand that this was flaky and didn't happen all the time, so it may take a while to confirm.

I'd recommend updating to the next stable version of `prisma` when it comes out, since `dev` versions are mostly for internal use and may contain unannounced breaking changes.

**Comment by Raamzeez** [comment]
@madebysid I have tried it recently and it appears that it is working! I am on the latest release with version 0.394.0, which ships with prisma version 2.23.0. All my data is shown when I open the application, although it admittedly takes a little longer than previous versions to show it. This is not an issue for me though, as I would prefer to have reliability over speed. Thanks for the help.
